app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -1256,6 +1356,26 @@ DeviceDelegateOpenXR::CreateLayerEquirect(const VRLayerPtr &aSource) {
   return result;
 }
 
+VRLayerPassthroughPtr
+DeviceDelegateOpenXR::CreateLayerPassthrough() {
+  if (!m.layersEnabled || !m.passthroughSupported || !m.passthrough) {
+    return nullptr;
+  }
+
+  VRLayerPassthroughPtr result = VRLayerPassthrough::Create();
+  if (m.passthroughLayer != nullptr) {
+    m.passthroughLayer->Destroy();
```

Comment:
Assign the pointer to XR_NULL_HANDLE

Reply:
I don't think we want that. Note that Destroy() destroys only the xrLayer, but not the main instance of OpenXRLayerPassthrough. The main instance is only created once in `CreateLayerPassthrough()`. I know it is a bit weird, but this is a result of having these wrappers and instantiating the layer from common code.

Reply:
Umm, then perhaps I'd suggest a different name for the "Destroy" method. WDYT ? When I read object->Destroy() it usually means a self-destruction logic. What about "DestroyXRLayer() ?

Reply:
Ok, that works for me.

Reply:
Again, thinking twice about this, the rest of layers also use Destroy() even though they don't actually destroy the instance itself. I think we should be consistent with the other layers here.

Reply:
Yeah I agree with @elima here, we can rename them all later if required.